docs/api/apiv3/endpoints/schemas.apib

```diff
@@ -118,17 +127,22 @@ The following excerpt examplifies the objects that can be found as a value of th
         "name": "Lorem ipsum",
         "required": true,
         "hasDefault": false,
-        "writable": false
+        "writable": false,
+        "location": "_links",
       }
     },
     "4": {}
   }
 }
 ```
 
-Given the example above, the client has to add the property `loremIpsum` to the schema if the depending property is `1`, `2` or `3`. If it is `4` the property does not exist. The property will not be writable if the value is `3`. The values allowed to be set differ between having `1` or `2` selected for the depending property.
+Given the example above, the client has to add the property `loremIpsum` to the schema if the depending property is `1`, `2` or `3`.
+If it is `4` the property does not exist.
+The property will not be writable if the value is `3`.
+The values allowed to be set differ between having `1` or `2` selected for the depending property.
 
-Because of the limitation of JSON objects, all keys will be strings, even when the depending value is actually something different (e.g. Integer, Date). This is also true for resources in which case the url of the resource is used as the key.
+Because of the limitation of JSON objects,all keys will be strings, even when the depending value is actually something different (e.g. Integer, Date).
```

Comment:
```suggestion
Because of the limitation of JSON objects, all keys will be strings, even when the depending value is actually something different (e.g. Integer, Date).
```